oaknational/Oak-Web-Application
# LESQ-1030: Parent subjects utilised if exist

## Description

Music year: 1965

- Utilises parent lesson for curriculum links if the parent subject exists
- Creates function to convert subject title to string
- Amends test / fixture data

## Issue(s)

Fixes #LESQ-1030

## Acceptance Criteria:

- [x]  Child subjects should link to the equivalent parent subject page (for the example above you should be taken to secondary science aqa)

## How to test

1. Go to https://deploy-preview-2719--oak-web-application.netlify.thenational.academy
2. Click on all units should navigate to the correct pages, ks4 subjects with parent science go to science pages

## Checklist
